src/Makefile.am

```diff
@@ -18,7 +18,7 @@ else
 PTHREAD_DEF =
 endif
 
-PROTOBUF_VERSION = 32:1:0
+PROTOBUF_VERSION = 32:0:0
```

Comment:
I think the internal template needs to be updated so that the changes to this file are not synced out.

All the changes in the cmake/ directory can also be reverted because those should not be using the internal templates anymore. Let me send out a change to delete those templates and add the cmake/ files to the exclusion list.

Reply:
@dlj-NaN said something about Kokoro needing those Makefiles so we might have to copy them into Piper instead of simply deleting them.